pkg/tracing/otlp_trace.go

```diff
@@ -174,7 +176,24 @@ func (s *TraceServiceServer) process(
 				span := &mem[i]
 				initSpanFromOTLP(span, scope, otlpSpan)
 				span.ProjectID = project.ID
-				s.Consumer.AddSpan(ctx, span)
+
+				for _, event := range span.Events {
+					eventSpan := &Span{
+						Attrs: NewAttrMap(),
+					}
+					initEventFromHostSpan(eventSpan, event, span)
+
+					if eventSpan.IsLog() {
+						s.LogConsumer.AddSpan(ctx, eventSpan)
+					} else if eventSpan.IsEvent() {
+						s.EventConsumer.AddSpan(ctx, eventSpan)
+					} else {
+						s.SpanConsumer.AddSpan(ctx, eventSpan)
```

Comment:
Event can't become a span

Reply:
Should i remove this brach or log something like "span is neither log nor event"?

Reply:
Let's log something